pom.xml

```diff
@@ -779,21 +772,6 @@
         <!-- https://stackoverflow.com/a/4086207/12916 -->
         <artifactId>maven-antrun-plugin</artifactId>
         <executions>
-          <execution>
-            <id>configureJavaAgent</id>
-            <goals>
-              <goal>run</goal>
-            </goals>
-            <phase>initialize</phase>
-            <configuration>
-              <target>
-                <condition else="" property="jenkins.javaAgent" value="-javaagent:${org.mockito:mockito-core:jar}">
-                  <isset property="org.mockito:mockito-core:jar" />
-                </condition>
-              </target>
-              <exportAntProperties>true</exportAntProperties>
-            </configuration>
-          </execution>
```

Comment:
Reimplemented with Java code in https://github.com/jenkinsci/maven-hpi-plugin/pull/741.